Makefile

```diff
@@ -94,7 +94,6 @@ endif
 $(CSI_BIN): $(CSI_BIN_SRCS)
 	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) go build -ldflags '$(LDFLAGS_CSI)' -o $(abspath $@) $<
 	@touch $@
-	size $(abspath $@)
```

Comment:
Are you removing `size $(abspath $@)`?

Reply:
"size $(abspath $@)" was introduced in my first commit, which should not be checked in.
My second commit reverted it.